lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -195,10 +195,11 @@ def _cleanup(self) -> None:
         if self.platform_runbook.capture_libvirt_debug_logs:
             self._disable_libvirt_debug_log()
 
-        dmesg_output = self.host_node.tools[Dmesg].get_output(force_run=True)
-        dmesg_path = self.host_node.local_log_path / "dmesg.txt"
-        with open(str(dmesg_path), "w") as f:
-            f.write(dmesg_output)
+        if self.host_node.is_remote:
```

Comment:
With Lili's change, we introduced a new flat called "is_test_target". Please consider if it's applicable here.

Reply:
`host_node` will never be the test target. So, I think `is_remote` is correct here.